modules/regular_service_perimeter/main.tf

```diff
@@ -28,13 +28,15 @@ resource "google_access_context_manager_service_perimeter" "regular_service_peri
 
   status {
     restricted_services = var.restricted_services
+    resources           = [for item in var.resources_dry_run : can(regex("global/networks", item)) ? format("//compute.googleapis.com/%s", item) : format("projects/%s", item)]
```

Comment:
@cmalpe it should be `var.resources` instead of `var.resources_dry_run`.

Reply:
Hi @imrannayer I missed that part! Thanks for finding this error. I have corrected it and pushed the changes.